app/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,8 @@
   <%= header.with_navigation_item(text: current_user.name) %>
   <% if current_user.can_manage_others %>
     <%= header.with_navigation_item(text: t('.manage_users'), href: admin_manage_users_root_path) %>
-  <% elsif current_user.supervisor? && FeatureFlags.basic_user_roles.enabled? %>
+  <%end%>
```

Comment:
`<% end %>` (spaces!) - does the `elsif` need to go i.e. can both links be shown simultaneously?

Reply:
Ah I think hassan's ide doesn't format automatically 😅 And yes for user managers they would see both links on staging. Supervisors can't manage others so they would only see the performance tracking tab

Reply:
We just ran `bundle exec erblint app/views/ -a` and it's not catching it 🤔